api/v1/status.proto

```diff
@@ -8,6 +8,14 @@ option (extproto.equal_all) = true;
 
 import "google/protobuf/struct.proto";
 
+/**
+ * ReporterStatus indicates the Status of a resource according to each reporter
+ */
+message ReporterStatus {
+    // Mapping from reporter (Status.reported_by) to that reporter's Status
+    map<string, Status> statuses = 1;
```

Comment:
let's add a controller ID or hash to a status so we know who wrote it in the single controller case. perhaps we can grab the pod revision and use that from env? and also use it as the key to map here

Reply:
Per our offline discussion I've modified this to use the pod's namespace